langserver/server.go

```diff
@@ -147,138 +120,25 @@ func ServerFromStream(ctx context.Context, stream jsonrpc2.Stream, config *Confi
 
 	s.lifetime, s.exit = context.WithCancel(ctx)
 
+	// it is safe to not check the error at this point,
+	// because the error will always be nil when an url is empty
+	s.prometheusClient, _ = promClient.NewClient("") // nolint: errcheck
```

Comment:
In that case it still makes sense to check the error and panic in case there is an error.

Reply:
done